janusgraph-dist/pom.xml

```diff
@@ -601,41 +593,10 @@
                                         <failIfNoTests>false</failIfNoTests>
                                         <excludes>
                                             <exclude>**/*HBase*IT.java</exclude>
-                                            <exclude>**/*Cassandra*IT.java</exclude>
```

Comment:
Why did you remove it?

Reply:
I removed also both classes which are matched by this.